src/main/java/lotto/ui/Printer.java

```diff
@@ -0,0 +1,116 @@
+package lotto.ui;
+
+import lotto.domain.NumberOfLottoTicket;
+import lotto.domain.WinningStatus;
+import lotto.domain.lotto.LottoAutomaticTicket;
+import lotto.domain.lotto.LottoAutomaticTickets;
+import lotto.domain.lotto.LottoManualTicket;
+import lotto.domain.lotto.LottoManualTickets;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+
+public class Printer {
+    private static final String REQUEST_PURCHASE_AMOUNT_MESSAGE = "구입금액을 입력해 주세요.";
+    private static final String REQUEST_LOTTO_MANUAL_TICKET_NUMBER_MESSAGE = "수동으로 구매할 로또 수를 입력해 주세요.";
+    private static final String REQUEST_LOTTO_MANUAL_TICKET_MESSAGE = "수동으로 구매할 번호를 입력해 주세요.";
+    private static final String PRINT_NUMBER_OF_LOTTO_TICKET_MESSAGE = "수동으로 %d장, 자동으로 %d개를 구매했습니다.";
+    private static final String REQUEST_LAST_WEEK_LOTTO_WINNING_NUMBER_MESSAGE = "\n지난 주 당첨 번호를 입력해 주세요.";
+    private static final String REQUEST_LOTTO_BONUS_BALL_NUMBER_MESSAGE = "보너스 볼을 입력해주세요";
+    private static final String PRINT_FINAL_MATCHED_LOTTO_RESULT_MESSAGE = "%s개 일치%s(%s원)- ";
+    private static final String PRINT_MATCH_BONUS_BALL_MESSAGE = ", 보너스 볼 일치";
+    private static final String PRINT_PROFIT_LOSS_MESSAGE = "(기준이 1이기 때문에 결과적으로 손해라는 의미임)";
+    private static final String PRINT_LOTTO_PROFIT_MESSAGE = "총 수익률은 %.2f입니다.";
+
+    public void requestPurchaseAmount() {
+        System.out.println(REQUEST_PURCHASE_AMOUNT_MESSAGE);
+    }
+
+    public void requestLottoManualTicketNumber() {
+        System.out.println(REQUEST_LOTTO_MANUAL_TICKET_NUMBER_MESSAGE);
+    }
+
+    public void requestLottoManualTicket() {
+        System.out.println(REQUEST_LOTTO_MANUAL_TICKET_MESSAGE);
+    }
+
+    public void printNumberOfEachLottoTicket(NumberOfLottoTicket numberOfLottoTicket) {
+        System.out.println(String.format(
+                PRINT_NUMBER_OF_LOTTO_TICKET_MESSAGE,
+                numberOfLottoTicket.getNumberOfManualLottoTicket(),
+                numberOfLottoTicket.getAutomaticLottoTicketOfNumber()
+        ));
+    }
```

Comment:
```suggestion
    public void printNumberOfEachLottoTicket(NumberOfLottoTicket numberOfLottoTicket) {
        System.out.printf(
                PRINT_NUMBER_OF_LOTTO_TICKET_MESSAGE + '%n',
                numberOfLottoTicket.getNumberOfManualLottoTicket(),
                numberOfLottoTicket.getAutomaticLottoTicketOfNumber()
        );
    }
```

`System.out.printf`를 사용하면 Sring.format을 사용하지 않고 서식을 이용하여 출력할 수 있습니다